app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -206,7 +207,11 @@ public void onValueChanged(@NonNull Boolean loading) {
                 }
 
                 final MainActivity activity = (MainActivity)getActivity();
-                activity.setCursorEnabled(!getWebView().getUrl().contains("youtube.com/tv"));
+                final IWebView webView = getWebView();
+                // Enable the cursor if website is youtube.com/tv
+                final boolean enableCursor = webView == null ? false :
```

Comment:
nit: I think nested ternaries throw up a red flag in my mind but I'm not sure if it's warranted.

I think it's just my caution to use ternaries carefully because they can be confusing (since they're just a garble of syntax).